releasenotes/README.md

```diff
@@ -12,13 +12,25 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
 
 [Click here for the v50 release notes.](./v50.md)
 
-## April 29, 2021
+## 51.10.0 (April 29, 2021) - CLI 7.99.0
 
-These changes are in the release candidate plug-in (`salesforcedx@latest-rc`). We plan to include these changes in next week's official release. This list isn't final and is subject to change.
+* NEW: Use a TAR file to install the Salesforce CLI release candidate. The download URL is similar to the URL for installing the current release, but uses the `stable-rc` channel instead of `stable`. For example, use this `wget` command to get the Linux release candidate TAR file:
```

Comment:
```suggestion
* NEW: Use a `.tar` file to install the Salesforce CLI release candidate. The download URL is similar to the URL for installing the current release, but uses the `stable-rc` channel instead of `stable`. For example, use this `wget` command to get the Linux release candidate `.tar` file:
```

Reply:
If you want to follow CCX styles, it's in mono font, with .xxx (l/c)

Reply:
Thx for the reminder.  I personally don't like that style, so I always "forget" it. :)